lib/orwell/plug/github.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Orwell.Plug.GitHub do
+  @moduledoc """
+  Defines a helper to provide easy access to a GitHub client instance
+  based on the currently logged in user
+  """
+
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    current_user = Guardian.Plug.current_resource(conn)
+    token = current_user.github_credentials["token"]
+    config = Orwell.GitHub.Config.new(token)
+
+    conn
+    |> assign(:config, config)
```

Comment:
how about renaming this to `:github_config`? the current name is a bit out of context in the controllers